Assets/Scripts/UserInterface/GameUI/LevelUpRewardScreenController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class LevelUpRewardScreenController : UIController {
+    [SerializeField]
+    GameObject LevelUpScreen;
+
+    public static LevelUpRewardScreenController Instance;
+
+    void Start()
+    {
+        Instantiate(LevelUpScreen);
```

Comment:
Can you make sure the LevelUp screen is hidden by default (should not be visible in the scene when the game is not running)